### ISSUE TITLE
Do I need to install chai?

### ISSUE THREAD

**Issue by samselikoff** [emberjs/ember-mocha#272]
I'm diagnosing a Mirage issue for a Mocha user and trying to get a demo app setup with `ember-mocha`. I followed the README and am trying to get the sample test working:

```js
import { expect } from 'chai';
import { describe, it } from 'mocha';
import { setupTest } from 'ember-mocha';

describe('SidebarController', function() {
  setupTest();

  // Replace this with your real tests.
  it('exists', function() {
    let controller = this.owner.lookup('controller:sidebar');
    expect(controller).to.be.ok;
  });
});
```

I'm getting an error

> Could not find module `chai` imported from...

Do I need to install `chai` for this to work? Should it have been installed as part of `ember install ember-mocha`? I'm not familiar with mocha or chai but found this confusing. Happy to PR some docs if it would be helpful!

**Comment by Turbo87** [comment]
Yes, we need to port over the ember-cli-mocha blueprints here and deprecate the other package
